frontend/src/components/SignIn/SignIn.css

```diff
@@ -1,116 +1,21 @@
-:root {
-  --input-padding-x: .75rem;
-  --input-padding-y: .75rem;
-}
-
-html,
-body {
-  height: 100%;
-}
-
-body {
-  display: -ms-flexbox;
-  display: -webkit-box;
-  display: flex;
-  -ms-flex-align: center;
-  -ms-flex-pack: center;
-  -webkit-box-align: center;
-  align-items: center;
-  -webkit-box-pack: center;
-  justify-content: center;
-  padding-top: 40px;
-  padding-bottom: 40px;
-  background-color: #f5f5f5;
-}
-
-.form-signin {
-  width: 100%;
-  max-width: 400px;
-  padding: 15px;
-  margin: 0 auto;
-}
 
-.form-label-group {
+.signInPassword.form-group.row {
```

Comment:
You don't need to specify all the classes on an element. It's preferable to just style the class you're trying to operate on (`.signInPassword` here) or use an ID.

Reply:
Fixed on multiple css files